buildSrc/src/main/kotlin/conventions/publishing.gradle.kts

```diff
@@ -126,6 +126,9 @@ signing {
     if (signingKey.isPresent && signingPassword.isPresent) {
         logger.debug("[${project.displayName}] Signing is enabled")
         useInMemoryPgpKeys(signingKey.get(), signingPassword.get())
+        sign(publishing.publications["maven"])
+    } else {
+        logger.warn("[${project.displayName}] No Signing keys found, skipping signing configuration")
```

Comment:
Let's make both the logging statements `info`, since it's normal to publish to the local repository without signing, and Sonatype won't let us release without signatures anyway.